functions/src/CVE/index.ts

```diff
@@ -1,46 +1,89 @@
 import * as functions from 'firebase-functions';
 import * as express from 'express';
 import * as bodyParser from "body-parser";
+import * as admin from 'firebase-admin';
+import * as Enumerable from 'linq';
 
 const app = express();
 const main = express();
 
 main.use(app);
 main.use(bodyParser.json());
 
-
 export const getCVE = functions.https.onRequest(main);
 
 app.get('/cves', (request, response) => {
+  
     const bulletinID = request.query.bulletinid;
-    if (bulletinID !== null){
-      //TODO: call helper function to query for bulletin data 
-    }
     const SPLID = request.query.splid;
-    if (SPLID !== null){
-      //TODO: call helper function to query for spl data
-    }
     const SPLStart = request.query.splstart;
-    if (SPLStart !== null){
-      //TODO: call helper function to query for spl start data 
+    const CVEID = request.query.cveid;
+    const SPL1 = request.query.spl1;
+    const SPL2 = request.query.spl2;  
+
+    if (bulletinID !== null && bulletinID !== undefined){
+      bulletinIDHelper(String(bulletinID),response);
     }
-    const CVEID = request.query.cveid; 
-    if (CVEID !== null){
-      //TODO: call helper function for cve id data 
+    else if (SPLID !== null && SPLID !== undefined){
```

Comment:
you can simplify these by just doing if (SPLID), etc. See https://developer.mozilla.org/en-US/docs/Glossary/Truthy